### PR TITLE
Adding new NTC template

### PR DESCRIPTION
**Pull request by kapilrathi**
ISSUE TYPE
 - New Template Pull Request

COMPONENT
cisco_ios_show_platform, Cisco_IOS, show platform 

SUMMARY
Adding new NTC template for Show platform on IOS

### CODE REVIEW

File: tests/cisco_ios/show_platform/cisco_ios_show_platform.parsed [hunk 1]
```diff
@@ -0,0 +1,91 @@
+parsed_sample:
```
Comment (Sandeepsr) [inline tests/cisco_ios/show_platform/cisco_ios_show_platform.parsed]: ---

File: templates/index [hunk 2]
```diff
@@ -204,6 +204,7 @@ cisco_ios_show_boot.template, .*, cisco_ios, sh[[ow]] boot
 cisco_ios_show_vlan.template, .*, cisco_ios, sh[[ow]] vlan
 cisco_ios_show_vrf.template, .*, cisco_ios, sh[[ow]] vrf
 cisco_ios_dir.template,  .*, cisco_ios, dir
+cisco_ios_show_platform.template, .*, cisco_ios, sh[[ow]] plat[[form]]
```
Comment (Sandeepsr) [inline templates/index]: put in order of length

File: tests/cisco_ios/show_platform/cisco_ios_show_platform.parsed (hunk 1 above)
Comment (kapilrathi) [reply tests/cisco_ios/show_platform/cisco_ios_show_platform.parsed]: Resolved

File: templates/index (hunk 2 above)
Comment (kapilrathi) [reply templates/index]: changes made

**Pull request merged**